app/forms/supplementary/DeclarationAdditionalActors.scala

```diff
@@ -16,23 +16,23 @@
 
 package forms.supplementary
 
-import forms.MetadataPropertiesConvertable
 import play.api.data.Forms.{optional, text}
 import play.api.data.{Form, Forms}
-import play.api.libs.json.Json
+import play.api.libs.json.{JsValue, Json}
 import uk.gov.voa.play.form.ConditionalMappings.mandatoryIfNot
 
-case class DeclarationAdditionalActors(eori: Option[String], partyType: Option[String])
-    extends MetadataPropertiesConvertable {
+case class DeclarationAdditionalActors(eori: Option[String], partyType: Option[String]) {
+  implicit val writes = Json.writes[DeclarationAdditionalActors]
```

Comment:
I'm curious why you need implicit writes when you have defined implicit format in companion object

Reply:
Because the Json.toJson needs implicit writes